### PR TITLE
Add services page

### PR DESCRIPTION
**Pull request by proddata**
## Summary of the changes / Why this is an improvement
Add page describing our service offerings.

relates to https://github.com/crate/cloud/issues/1436

## Checklist

 - [x] Link to issue this PR refers to (if applicable): 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/reference/services.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+| Shared | S4     | up to 3 | 4 GiB    | 8 GiB to 1TiB |
+| Shared | S6     | up to 4 | 6 GiB    | 8 GiB to 1TiB |
+| Shared | S8     | up to 8 | 12 GiB   | 8 GiB to 1TiB |
+:::
+
+**Variable Performance** <br>
+Since your cluster will be sharing vCPUs with other clusters, the performance
+may vary depending on the overall load on the underlying machine. This 
+variability makes it less predictable compared to Dedicated Plans, where you
+have dedicated resources.
```
Comment (amotl) [inline docs/reference/services.md:90]:
```suggestion
variability makes it less predictable compared to Dedicated Plans, where your
database is running on dedicated resources.
```

File: docs/reference/services.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+**Fair-Use Principle** <br>
+The Shared Plan operates on a fair-use principle. All users are expected to use
```
Comment (amotl) [inline docs/reference/services.md:93]:
```suggestion
The Shared Plan operates on a fair-use principle. All users are expected to utilize
```

File: docs/reference/services.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+| Plan | Size   | vCPUs     | RAM      |  Storage  |
+|----|--------|-----------|----------| ---- |
+| Dedicated | CR1    | up to 15.75  | up to 63 GiB    | up to 72 TiB |
+| Dedicated | CR2    | up to 31.5  | up to 126 GiB   | up to 72 TiB |
+| Dedicated | CR3    | up to 63  | up to 252 GiB   | up to 72 TiB |
+| Dedicated | CR4    | up to 126  | up to 495 GiB   | up to 72 TiB |
+:::
+
+**Recommended Setup for High Availability**<br>
+While it's possible to start with just one node, for applications requiring high
```
Comment (amotl) [inline docs/reference/services.md:142]:
```suggestion
While it's possible to start with just one node, for applications needing high
```

File: docs/reference/services.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+availability and fault tolerance, we recommend using at least 3 nodes. This
+ensures that your data is safely replicated and that the cluster can handle
+node failures gracefully.
+
+## Enterprise
+
+For organizations with specialized requirements that go beyond our Shared and
```
Comment (amotl) [inline docs/reference/services.md:149]:
```suggestion
For organizations with specialized requirements that go beyond the Shared and
```

File: docs/reference/services.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+Dedicated Plans, CrateDB Cloud offers Enterprise solutions tailored to your
+specific needs. Our sales team and solutions engineers work closely with you to
+architect and deploy a custom cluster configuration, ensuring optimal
+performance, scalability, and security for your mission-critical applications.
+Whether you have stringent compliance mandates, complex integrations, or unique
```
Comment (amotl) [inline docs/reference/services.md:154]: New paragraph, for a better stage of the "final words" on this page. ;]
```suggestion

Whether you have stringent compliance mandates, complex integrations, or unique
```

File: docs/reference/services.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+scalability challenges, our Enterprise solutions provide the flexibility and
+expertise to meet your business objectives.
```
Comment (amotl) [inline docs/reference/services.md:156]:
```suggestion
expertise to meet your business objectives and technical requirements.
```

**Review by amotl — CHANGES_REQUESTED** [review]
I love the whole page, and added some suggestions at your disposal.

File: docs/reference/services.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+The Shared Plan operates on a fair-use principle. All users are expected to
+utilize the shared resources responsibly so that the system remains equitable
+and functional for everyone.
+
+:::{note}
+__*CRFREE__: This plan is aimed at new users who want to test and evaluate
+CrateDB Cloud and is perpetually free to use. Every user can deploy one free
+tier cluster in their organization without adding a payment method. Free tier
+clusters will be suspended if they are not used for 7 days, and deleted after
+7 more days of idleness. They cannot be scaled or changed.
```
Comment (joncombe) [inline docs/reference/services.md:102]: I feel "inactivity" is a better word that "idleness". Not a big issue, just a suggestion.

**Review by joncombe — APPROVED** [review]
Approved with just one comment.

**Pull request merged**